docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,120 @@
+# Provider Networks
+
+
+## Table of Contents
+- [Provider Networks](#provider-networks)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Common prerequisites](#common-prerequisites)
+  - [Considerations when creating provider networks](#considerations-when-creating-provider-networks)
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in a data center.
+Example of network types in this category include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters can be deployed on provider networks, so the installer won't need to create
+tenant networks nor floating IPs.
+More information can be found about provider networks terminology [here][1].
+
+Here is a basic architecture of one OCP cluster running on a provider network and another one
+on a tenant network:
+
+![OCP on a provider network](provider-network.png)
+
+
+## Prerequisites
+
+* The [Neutron service][2] is enabled and accessible through the [OpenStack Networking API][3].
+* The Neutroni service is configured with the [port-security and allowed-address-pairs][4] extensions so the installer can
+  add the `allowed_address_pairs` attribute to ports.
+
+
+## Considerations when creating provider networks
+
+* The same project that you use to install OpenShift must own the provider network and subnet.
+
+    It is important that the provider network and subnet are owned by the same project that will be used
+    to install OpenShift (from the clouds.yaml) and we don't want them to be owned by `admin` otherwise
+    it'll cause Terraform to fail creating the ports.
```

Comment:
```suggestion
The provider network and subnet must be owned by the project that is used to install OpenShift instead of `admin`.
If they are not, Terraform fails while creating ports. 
//The provider network and subnet are indicated in the clouds.yaml file.
```

Not sure if that commented part is right or not.